packages/ses/README.md

```diff
@@ -503,6 +502,153 @@ system generated other diagnostic information hidden in side tables. The tamed
 console uses these side tables to output more informative diagnostics.
 [Logging Errors](./src/error/README.md) explains the design.
 
+## Security claims and caveats
+
+Provided that the `ses` implementation and its trusted compute base are
+correct, we claim that a host program (Figure 1) can evaluate a guest program
+(`program`) in a compartment after `lockdown` and that the program:
+
+- will initially only have access to one mutable object, `globalThis`,
+- specifically cannot modify any primordial object that it shares with the host
```

Comment:
```suggestion
- specifically cannot modify any primordial object that it shares with the host, except that compartment's `globalThis`, which the host has access to.
```

Reply:
I’m reading `globalThis` as not a shared primordial, for not being shared between compartments and also not primordial since it’s compartment-specific. I think the prior bullet makes that clear.